moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java

```diff
@@ -142,7 +142,7 @@ public List<String> readFile(String file) throws IOException {
 				if (sb == null) {
 					sb = new StringBuilder();
 				}
-				sb.append(nextLine).append("\r\n");
+				sb.append(nextLine).append("\r");
```

Comment:
Can we make this a constant LINE_TERMINATOR in case it ever needs to be modified again?